python/pylibcugraph/pylibcugraph/_cugraph_c/community_algorithms.pxd

```diff
@@ -104,6 +104,18 @@ cdef extern from "cugraph_c/community_algorithms.h":
             cugraph_error_t** error
         )
     
+    cdef cugraph_error_code_t \
```

Comment:
I should mark the above function (the old `cugraph_louvain`) as deprecated.  Not sure how to annotate that correctly.

Reply:
I believe you can raise `DeprecationWarning` from the cython .pyx code for this.  Although, don't we want the cleaner `cugraph_louvain` function name longer term? `cugraph_louvain_with_threshold` seems clunkier.

Reply:
+1 for making a breaking change if we are not aware of any users.

Reply:
I'm also a +1 for a breaking change to keep the better API name without the process required above, esp. since we're not aware of any users.

Reply:
Made this change.